src/main/java/com/yelp/nrtsearch/server/luceneserver/search/collectors/MyTopSuggestDocsCollector.java

```diff
@@ -70,6 +70,17 @@ public void fillHitRanking(SearchResponse.Hit.Builder hitResponse, ScoreDoc scor
     if (!Float.isNaN(scoreDoc.score)) {
       hitResponse.setScore(scoreDoc.score);
     }
+    if (scoreDoc instanceof TopSuggestDocs.SuggestScoreDoc) {
+      // set the retrieval key from the SuggestScoreDoc to the hit response as "__suggest_key" field
+      hitResponse.putFields(
+          "__suggest_key",
```

Comment:
nit: can you make this a static final class constant

Reply:
done